### PR TITLE
Add test for unzip

### PR DESCRIPTION
**Pull request by drpaneas**
Test for extraction in PWD.

This adds the first test requested by poo#15470.

Example run can be found [here](http://dede.arch.suse.de/tests/314#)

### CODE REVIEW

File: tests/console/unzip.pm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+sub run() {
+
+    select_console "root-console";
+    assert_script_run "mkdir -p /tmp/unzip-test/";
+
+    # Basic Usage (Extract an archive with 7 files and check if all of them were extracted)
+    assert_script_run "wget --quiet " . data_url('console/test_unzip.zip') . " -O /tmp/unzip-test/archive.zip";
+    assert_script_run "cd /tmp/unzip-test; unzip /tmp/unzip-test/archive.zip";
+    my $entries = script_output("ls -l /tmp/unzip-test/ | wc -l");
+    die "Extract produced too few values: $entries instead of 7" unless ($entries eq "7");
```
Comment (coolo) [inline tests/console/unzip.pm]: I would suggest to go one step further and not ls, but 
md5sum /tmp/unzip-test/*  | sort and verify the output of *that*

File: tests/console/unzip.pm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+use testapi;
+
+sub run() {
+
+    select_console "root-console";
+    assert_script_run "mkdir -p /tmp/unzip-test/";
+
+    # Basic Usage (Extract an archive with 7 files and check if all of them were extracted)
+    assert_script_run "wget --quiet " . data_url('console/test_unzip.zip') . " -O /tmp/unzip-test/archive.zip";
+    assert_script_run "cd /tmp/unzip-test; unzip /tmp/unzip-test/archive.zip";
```
Comment (kravciak) [inline tests/console/unzip.pm]: Unzip should not need absolute path after cd

File: tests/console/unzip.pm [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    my $entries = script_output("ls -l /tmp/unzip-test/ | wc -l");
```
Comment (kravciak) [inline tests/console/unzip.pm]: You can also use validate_script_output, but it's up to your preference..

**Comment by okurz** [comment]
please rebase on current master and make sure your "os-autoinst" subfolder is up to date to use an updated style check. You may have to create a symlink to `.perltidyrc` manually, e.g. with `ln -s os-autoinst/.perltidyrc`

**Pull request merged**